http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -472,14 +474,24 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       // Invoke helm
       _ <- helmClient
         .installChart(
-          org.broadinstitute.dsp.Release(config.galaxyAppConfig.releaseName.value),
+          org.broadinstitute.dsp.Release(releaseName.value),
           org.broadinstitute.dsp.Chart(config.galaxyAppConfig.chart.value),
           org.broadinstitute.dsp.Values(chartValues)
         )
         .run(helmAuthContext)
 
+      // Poll galaxy until it starts up
+      _ <- (
+        Stream.sleep_(config.monitorConfig.createApp.interval) ++
+          Stream.eval(
+            galaxyDAO.isProxyAvailable(dbCluster.googleProject, appName)
+          )
+      ).repeatN(config.monitorConfig.createApp.maxAttempts)
+        .takeThrough(_ == false)
+        .compile
+        .lastOrError
```

Comment:
This pattern looks nice and seems to be utilized in a few places so it may be nice to factor it out into a util method that accepts parameters for `Stream.eval()`, `repeatN()` and `takeThrough()`.

Reply:
Maybe… although I feel like the fs2 combinators are already pretty general and easy to read, not sure a util method would make things any simpler.

Reply:
you can use `streamFUntilDone` directly I think... you just need to provide a `DoneCheckable[Boolean]` instance

Reply:
you could use streamFUntilDone https://github.com/broadinstitute/workbench-libs/blob/develop/google2/src/main/scala/org/broadinstitute/dsde/workbench/google2/package.scala#L121